crates/bevy_gltf/src/lib.rs

```diff
@@ -58,3 +60,9 @@ pub struct GltfPrimitive {
     pub mesh: Handle<Mesh>,
     pub material: Option<Handle<StandardMaterial>>,
 }
+
+#[derive(Debug, Reflect, Default)]
```

Comment:
```suggestion
#[derive(Clone, Debug, Default, Reflect)]
```

Reply:
I don't see why `Clone` shouldn't be supported (unless there is a specific reason) :)